stage1/init/pod.go

```diff
@@ -168,10 +160,10 @@ func (p *Pod) appToSystemd(ra *schema.RuntimeApp, interactive bool) error {
 	}
 
 	env := app.Environment
-	env.Set("AC_APP_NAME", name)
+	env.Set("AC_APP_NAME", appName.String())
```

Comment:
@jonboulle For example [[1]](https://github.com/coreos/rkt/blob/master/store/aciinfo.go#L30) [[2]](https://github.com/coreos/rkt/blob/master/store/schema.go#L40)